tmt/steps/provision/bootc.py

```diff
@@ -135,6 +135,15 @@ class BootcData(tmt.steps.provision.testcloud.ProvisionTestcloudData):
              building the bootc disk image.
              """)
 
+    rootfs: str = field(
+        default="xfs",
+        option=('--rootfs'),
+        metavar='ROOTFS',
+        help="""
+             Select root filesystem type. Overrides the default from the source 
+             container. Supported value: ext4, xfs, and btrfs
+             """)
```

Comment:
Two usability hints:

* if there is a set of choices, please use `choices=[...]` to list them, see e.g. https://github.com/teemtee/tmt/blob/main/tmt/cli/_root.py#L221
* do not mention the choices in the help text, adding `choices=` will make them appear as metavar, i.e. `--rootfs=xfs|ext4|btrfs`, which means you should also drop `metavar`.